Cargo.toml

```diff
@@ -18,8 +18,12 @@ exclude = [
     ".gitignore",
     ".travis.yml",
 ]
+build = "build.rs"
 
 [dependencies]
-python27-sys="*"
 libc="*"
 num="*"
+
+# TODO: crates.io version doesn't have necessary conditional cfg flags yet
+[dependencies.python27-sys]
+git = "https://github.com/dgrunwald/rust-cpython.git"
```

Comment:
Instead of referencing the git repository with `git = ...`, it's also possible to use `path = "python27-sys"` here.
But actually I think the best solution would be to revert to `python27-sys="*"`, and rely on the cargo path override (to be added to version control) during development. Then we just need to make sure to release python27-sys before releasing a new cpython package.